sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.institutions.controller.ts

```diff
@@ -63,4 +68,92 @@ export class ApplicationOfferingChangeRequestInstitutionsController extends Base
       count: applications.count,
```

Comment:
I apologize I missed it in the previous PR but can we also include that the count of applications is being returned too along with the list of eligible applications in the return part of the method comment.